controllers/openshift_controller.go

```diff
@@ -631,60 +631,64 @@ func (r *KataConfigOpenShiftReconciler) createScc() error {
 	return nil
 }
 
-func (r *KataConfigOpenShiftReconciler) createRuntimeClass(runtimeClassName string, cpuOverhead string, memoryOverhead string) error {
-
-	rc := func() *nodeapi.RuntimeClass {
-		rc := &nodeapi.RuntimeClass{
-			TypeMeta: metav1.TypeMeta{
-				APIVersion: "node.k8s.io/v1",
-				Kind:       "RuntimeClass",
-			},
-			ObjectMeta: metav1.ObjectMeta{
-				Name: runtimeClassName,
-			},
-			Handler: runtimeClassName,
-			// Use same values for Pod Overhead as upstream kata-deploy using, see
-			// https://github.com/kata-containers/packaging/blob/f17450317563b6e4d6b1a71f0559360b37783e19/kata-deploy/k8s-1.18/kata-runtimeClasses.yaml#L7
-			Overhead: &nodeapi.Overhead{
-				PodFixed: corev1.ResourceList{
-					corev1.ResourceCPU:    resource.MustParse(cpuOverhead),
-					corev1.ResourceMemory: resource.MustParse(memoryOverhead),
+type RuntimeClassConfig struct {
+	ClassName      string
+	CPUOverhead    string
+	MemoryOverhead string
+}
+
+func (r *KataConfigOpenShiftReconciler) createRuntimeClasses(runtimeClassConfigs []RuntimeClassConfig) error {
+	for _, config := range runtimeClassConfigs {
```

Comment:
Frankly, I'd prefer if this loop just called the original `createRuntimeClass()` function rather than inline it here...